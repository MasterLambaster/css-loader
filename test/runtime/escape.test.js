/* eslint-env mocha*/

const escape = require('../../src/runtime/escape');

describe('escape', () => {
  it('basic', () => {
    expect(escape(true)).toMatchSnapshot(true);
    expect(escape('image.png')).toMatchSnapshot();
    expect(escape('"image.png"')).toMatchSnapshot();
    expect(escape("'image.png'")).toMatchSnapshot();
    expect(escape('image other.png')).toMatchSnapshot();
    expect(escape('"image other.png"')).toMatchSnapshot();
    expect(escape("'image other.png'")).toMatchSnapshot();
    expect(escape('image"other.png')).toMatchSnapshot();
    expect(escape('image\nother.png')).toMatchSnapshot();
  });
});
