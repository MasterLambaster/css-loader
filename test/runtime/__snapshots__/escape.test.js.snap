// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`escape basic 1`] = `true`;

exports[`escape basic 2`] = `"image.png"`;

exports[`escape basic 3`] = `"image.png"`;

exports[`escape basic 4`] = `"image.png"`;

exports[`escape basic 5`] = `"\\"image other.png\\""`;

exports[`escape basic 6`] = `"\\"image other.png\\""`;

exports[`escape basic 7`] = `"\\"image other.png\\""`;

exports[`escape basic 8`] = `"\\"image\\\\\\"other.png\\""`;

exports[`escape basic 9`] = `"\\"image\\\\nother.png\\""`;
